### PR TITLE
LCI 10 cards

### PR DESCRIPTION
**Pull request by Simisays**
some of these uncommons are more complicated that I expected

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/join_the_dead.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Join the Dead
+ManaCost:1 B B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ -5 | NumDef$ -5 | IsCurse$ True | ConditionCheckSVar$ X | ConditionSVarCompare$ LT4 | SubAbility$ PumpDescended | SpellDescription$ Target creature gets -5/-5 until end of turn. Descend 4 — That creature gets -10/-10 until end of turn instead if there are four or more permanent cards in your graveyard.
+SVar:PumpDescended:DB$ Pump | Defined$ Targeted | NumAtt$ -10 | NumDef$ -10 | ConditionCheckSVar$ X | ConditionSVarCompare$ GE4
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/join_the_dead.txt:5]: I think Compare SVar will be a bit cleaner

File: forge-gui/res/cardsfolder/upcoming/kitesail_larcenist.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Kitesail Larcenist
+ManaCost:2 U
+Types:Creature Human Pirate
+PT:2/3
+K:Flying
+K:Ward:1
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigAnimate | TriggerDescription$ When CARDNAME enters the battlefield, for each player, choose up to one other target artifact or creature that player controls. For as long as CARDNAME remains on the battlefield, the chosen permanents become Treasure artifacts with "{T}, Sacrifice this artifact: Add one mana of any color" and lose all other abilities.
+SVar:TrigAnimate:DB$ Animate | RemoveAllAbilities$ True | Types$ Artifact,Treasure  | Duration$ UntilHostLeavesPlay | Abilities$ TreasureSac | RemoveCardTypes$ True | Duration$ Permanent | ValidTgts$ Artifact,Creature | TgtPrompt$ Select up to one target artifact or creature that player controls | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kitesail_larcenist.txt:8]: remove wrong Duration

File: forge-gui/res/cardsfolder/upcoming/malamet_battle_glyph.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Name:Malamet Battle Glyph
+ManaCost:G
+Types:Sorcery
+A:SP$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature you control | CounterType$ P1P1 | CounterNum$ 1 | ConditionDefined$ Targeted | ConditionPresent$ Creature.ThisTurnEntered | SubAbility$ DBFight | SpellDescription$ Choose target creature you control and target creature you don't control. If the creature you control entered the battlefield this turn, put a +1/+1 counter on it. Then those creatures fight each other.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/malamet_battle_glyph.txt:4]: `ConditionDefined$ ThisTargetedCard` so you don't check the wrong one...

File: forge-gui/res/cardsfolder/upcoming/join_the_dead.txt (hunk 1 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/join_the_dead.txt:5]: yeah it would be cleaner as one Pump effect with Variable values

**Review by Hanmac — APPROVED** [review]

**Pull request merged**